### PR TITLE
File Checkpoint History View

### PR DESCRIPTION
**Pull request by marsha88**
This PR just contains the UI components for listing a file's history (the checkpoint message, and the date of committing)

### CODE REVIEW

File: app/actions/action_types.js [hunk 1]
```diff
@@ -1 +1 @@
-export const FORM_SUBMIT_SUCCEEDED = '@@redux-form/SET_SUBMIT_SUCCEEDED';
+export const FORM_SUBMIT_SUCCEEDED = "@@redux-form/SET_SUBMIT_SUCCEEDED";
```
Comment (drewatk) [inline app/actions/action_types.js]: The pattern I was following was `app/actions/form.js` which would export this. If you wans to do this then move my action types to this file

File: app/components/History/CheckpointView.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Card, CardBody, CardSubtitle, CardText } from "reactstrap";
+
+const CheckpointView = ({ message, date }) => (
```
Comment (drewatk) [inline app/components/History/CheckpointView.js]: Call this `CheckpointCard`

File: app/reducers/checkpoint_history.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const now = new Date().toLocaleString("en-US", { timeZone: "UTC" });
+
+const defaultHistory = [
+  { message: "first commit", date: now },
+  { message: "second commit", date: now },
+  { message: "day 68. still doesn't work. i'm giving up", date: now }
```
Comment (drewatk) [inline app/reducers/checkpoint_history.js]: 😢

File: app/reducers/checkpoint_history.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+];
+
+const checkpointHistory = (state = defaultHistory) => state;
```
Comment (drewatk) [inline app/reducers/checkpoint_history.js]: Is this final? what does this reducer effectively do then?

(asking because I really don't know)

Comment (marsha88) [reply app/reducers/checkpoint_history.js]: No. This reducer will be eventually be used to update checkpoint state when we change the current file and grab its commit history.

**Pull request merged**